gallery/src/components/NxTag/NxTagExample.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTag } from '@sonatype/react-shared-components';
+
+function NxTagExample() {
+  return (
+    <>
+      <NxTag>Default</NxTag>
+      <NxTag color="purple">Purple</NxTag>
+      <NxTag color="pink">Pink</NxTag>
```

Comment:
Probably should have kept this long to demonstrate the 320px max width

Reply:
> Probably should have kept this long to demonstrate the 320px max width

https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/d032f8773dc3e861ffec22ea797ccdaf8b8108b6